src/osx/cocoa/textctrl.mm

```diff
@@ -1111,13 +1122,17 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
                                 long end,
                                 const wxTextAttr& style)
 {
+    NSMutableParagraphStyle *paragraphStyle;
+    NSTextStorage *storage;
+    NSRange range;
```

Comment:
Are you sure this `range` is actually used? This code is very confusing because it uses the same variable in different places for apparently different things.